frontend.sh

```diff
@@ -93,6 +93,17 @@ build() {
   fi
 }
 
+# Run tasks to build all project assets for production distribution.
+dry_run() {
+  set DJANGO_SETTINGS_MODULE=cfgov.settings.production
+  set DJANGO_STATIC_ROOT=$(pwd)/collectstatic
+  pip install -r requirements/optional-public.txt
+  pip install -r requirements/optional-private.txt
```

Comment:
It occurs to me that we *might* want to do this in a tox environment to avoid muddying up someone's working env.

Reply:
yeah, how would you do that outside a testing environment?

Reply:
Create a testing environment in tox.ini :) 

Unfortunately to do so would require `tox` to be installed, which I don't think we can expect to have when we get to `frontend.sh`. 

Happy to help figure this out if you like.

Reply:
ok, I've added a script to tox.ini but I'm now getting a `No module named MySQLdb` error. I'm not sure why MySQLdb would be necessary when running collectstatic. Also, it's hecka slow and the `-vv` flag doesn't work as expected unless we upgrade to tox 2.9.

Reply:
ok, I just added mysql as a dependency. Should be good to go now.